app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,63 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    private HashMap<Long, String> tasks = new HashMap<>();
```

Comment:
```suggestion
    private final HashMap<Long, String> tasks = new HashMap<>();
```

불변이라면 final은 항상 붙인다고 생각하시면 좋습니다!